### DISCUSSION TITLE
Enhancement: Organize debate contribution in folders

### DISCUSSION THREAD

**Discussion by zukunft** [orbiting/republik-frontend#546] [Ideas]
As the number of debate contributions are increasing it gets more and more difficult to find older contributions to use them as a reference. This can be improved by allowing to create folders.

**Comment by tpreusse** [comment]
How do you imagine those folders? Do you want to save your own or also comments from others? Would it be like a bookmark list with articles and comments?

**Comment by zukunft** [comment]
Yes, I want to be able to create a folder and drop my contributions to it. E.g. I have made many contributions related to COVID, but many of these contributions are now not relevant any more, because the situation has changed. Other contributions have more a long term target and I an referring to these contributions from time to time. With all the outdated COVID contributions the long term are sometimes hard to find.

P.S. I have successfully installed the front end on my local PC and I am now trying to setup the back end. The target hat been that I try to solve the issues created by me. But maybe this takes a while ...
